src/wolfgame.py

```diff
@@ -1800,33 +1799,23 @@ def chk_decision(cli, force=""):
         # we only need 50%+ to not lynch, instead of an actual majority, because a tie would time out day anyway
         # don't check for ABSTAIN_ENABLED here since we may have a case where the majority of people have pacifism totems or something
         if len(not_lynching) >= math.ceil(avail / 2):
-            abs_evt = Event("chk_decision_abstain", {})
+            abs_evt = Event("chk_decision_abstain", {}, votelist=votelist, numvotes=numvotes)
             abs_evt.dispatch(cli, var, not_lynching)
             cli.msg(botconfig.CHANNEL, messages["village_abstain"])
-            if var.ROLES["succubus"] & var.NO_LYNCH:
-                var.ENTRANCED_DYING.update(var.ENTRANCED - var.NO_LYNCH - var.DEAD)
-            else:
-                var.ENTRANCED_DYING.update(var.ENTRANCED & var.NO_LYNCH)
             var.ABSTAINED = True
             event.data["transition_night"](cli)
             return
         for votee, voters in votelist.items():
-            # split this into a priority 0 event when it comes time to split off succubus
-            if votee in var.ROLES["succubus"]:
-                for vtr in var.ENTRANCED:
-                    if vtr in voters:
-                        voters.remove(vtr)
-
             if numvotes[votee] >= votesneeded or votee == force:
                 # priorities:
                 # 1 = displaying impatience totem messages
                 # 3 = mayor/revealing totem
                 # 4 = fool
                 # 5 = desperation totem, other things that happen on generic lynch
-                vote_evt = Event("chk_decision_lynch", {
-                    "votee": votee,
-                    "deadlist": deadlist
-                    }, del_player=del_player)
+                vote_evt = Event("chk_decision_lynch", {"votee": votee, "deadlist": deadlist},
+                    del_player=del_player, original_votee=votee,
+                    force=(votee == force), votelist=votelist,
+                    not_lynching=not_lynching)
```

Comment:
You shouldn't wrap arguments like this according to PEP 8. It's harder to follow.